test/e2e/e2e.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Is "The Kubernetes Authors" intended for general use by any project or just Kubernetes itself? Also is this header needed as the existing files in this project don't have it?

Reply:
oops, I'll change that.